packages/fluent-theme/src/fluent/FluentTheme.ts

```diff
@@ -8,14 +9,25 @@ export const FluentTheme: ITheme = createTheme({
     neutralPrimary: NeutralColors.gray160,
     neutralPrimaryAlt: NeutralColors.gray150,
     neutralSecondary: NeutralColors.gray130,
+    neutralSecondaryAlt: NeutralColors.gray110,
     neutralTertiary: NeutralColors.gray90,
     neutralTertiaryAlt: NeutralColors.gray60,
     neutralQuaternary: NeutralColors.gray50,
     neutralQuaternaryAlt: NeutralColors.gray40,
     neutralLight: NeutralColors.gray30,
     neutralLighter: NeutralColors.gray20,
     neutralLighterAlt: NeutralColors.gray10,
-    white: NeutralColors.white
+    white: NeutralColors.white,
+    // Shared Colors
+    red: SharedColors.red10,
```

Comment:
these only need to be added if they're different than the default

Reply:
By these, you mean the red colors or the white too? cause technically the white and black are the same 😄

Reply:
white and black too! might as well keep it slim

Reply:
Will clean the white and black but the reds are differing from the default colors so I need these ones in here.